tests/README.md

```diff
@@ -43,162 +42,42 @@ the baseline data are computed and sent to standard output. For this
-   libip_8.a -> $root/ip/vX.Y.Z/libip_vX.Y.Z_8.a   (double precistion)
-
-So if that is how IPOLATES was built in step (1), there is
-nothing more to do. 
-
-However, if you want to build with another version of the library,
-or if you built them in step (1) using the "gnu" or "nco_cray" install option,
-you will need to adjust these links.  Example: Suppose you want to
-test with the OPS version of IPOLATES on WCOSS Phase 2.  You would set the
-links as follows:
-
-  incmod_4 -> /nwprod2/lib/ip/vX.Y.Z/include/ip_vX.Y.Z_4
-  incmod_d -> /nwprod2/lib/ip/vX.Y.Z/include/ip_vX.Y.Z_d
-  incmod_8 -> /nwprod2/lib/ip/vX.Y.Z/include/ip_vX.Y.Z_8
- 
-  libip_4.a -> /nwprod2/lib/ip/vX.Y.Z/libip_vX.Y.Z_4.a
-  libip_d.a -> /nwprod2/lib/ip/vX.Y.Z/libip_vX.Y.Z_d.a
-  libip_8.a -> /nwprod2/lib/ip/vX.Y.Z/libip_vX.Y.Z_8.a
-
-
-3) Build the "scalar" and "vector" unit test programs 
-by running the "make_unit_test.ksh" build script.  (Use the 
-"make_unit_test_wcoss-cray.ksh" build script on WCOSS-Cray.)
-
-The IPOLATES library is dependant on the NCEP SP 
-(spectral) library (SPLIB).  Intel compiled versions of SPLIB
-exist on Theia and WCOSS Phase 1/2.  When running "make_unit_test.ksh"
-on these machines, these SP libraries are automatically located
-by module loads and stored in these environment variables:
-
- SP_LIB4 (single precision)
- SP_LIBd (mixed precision)
- SP_LIB8 (double precistion)
-
-On WCOSS-Cray, Intel and Cray compiled versions of SPLIB exist.
-And the "make_unit_test_wcoss-cray.ksh" script will locate
-them by module loads. 
-
-If you are testing on a machine other than WCOSS or Theia, 
-or you have compiled IPOLATES with a compiler other than Intel or Cray,
-you will need to locate (or build) SPLIB and set these environment
-varibles by editing the build script, or by exporting these
-variables on the command line before invoking it.
-Example (when using bash):
-
- $ export SP_LIB4=$directory/libsp_4.a
-
-Once the SP libraries have been located, run "make_unit_test.ksh" 
-(or make_unit_test_wcoss-cray.ksh) from the command line.  The script 
-takes one argument: the "config-setup" file that was used to compile
-IPOLATES. So, if you built IPOLATES with the Intel-optimized option, 
-you would invoke as follows:
-
- $ make_unit_test.ksh ifort.setup
-
-On WCOSS-Cray, you would do the following:
-
- $ make_unit_test_wcoss-cray.ksh ifort.setup
-
-
-4) Run the unit test.  Go to ./scripts.  To run interactively, type:
-
- $ run_unit_test.ksh
-
-You may want to pipe the output to a log fle.  Check output for any 
-"large" differences between the computed and baseline data.
-
-To run "run_unit_test.ksh" on the WCOSS or Theia compute nodes,
-invoke these driver scripts as follows:
-
- $ run_wcoss.ksh       (WCOSS Phase 1/2)
- $ run_wcoss-cray.ksh  (WCOSS-Cray)
- $ run_wcoss-dell.ksh  (WCOSS-Phase 3/Dell)
- $ sbatch run_theia.ksh  (Theia)
+From the build directory simply run `make test` to run all the tests
```

Comment:
Well that'a a lot simpler!

Reply:
I agree.  It is very easy to run these tests.